Modules/DataModel/Sources/PocketCastsDataModel/Private/DB/GRDB/GRDBDatabase.swift

```diff
@@ -17,7 +18,12 @@ class GRDBDatabase: PCDatabase {
     }
 
     func executeUpdate(_ sql: String, values: [Any]?) throws {
-        try database.execute(sql: sql, arguments: StatementArguments(values != nil ? values! : [])!)
+        // We want GRDB to save `Date` the same way FMDB does: using `timeIntervalSince1970`
+        // In terms of raw performance changing that on the model layer would be much better.
+        // However, that's a large change that we want to avoid.
+        let filteredValues = values?.map { ($0 as? Date)?.timeIntervalSince1970 ?? $0 }
+
+        try database.execute(sql: sql, arguments: StatementArguments(filteredValues != nil ? filteredValues! : [])!)
```

Comment:
Do we want to keep the force unwrapping for StatementArguments? 
Is this for us to detect any arguments that is not `DatabaseValueConvertible` ?

Reply:
I would keep this way for now.

You can see a good example described in this comment: https://github.com/Automattic/pocket-casts-ios/pull/2840#issuecomment-2787429208

tl;dr: forcing unwrapping for GRDB made me discover underlying issues on our unit tests.